lib/core/utils/scroll-state.js

```diff
@@ -14,13 +14,16 @@ function setScroll(elm, top, left) {
  * Create an array scroll positions from descending elements
  */
 function getElmScrollRecursive(root) {
-	return Array.from(root.children).reduce((scrolls, elm) => {
-		const scroll = axe.utils.getScroll(elm);
-		if (scroll) {
-			scrolls.push(scroll);
-		}
-		return scrolls.concat(getElmScrollRecursive(elm));
-	}, []);
+	return Array.from(root.children || root.childNodes || []).reduce(
```

Comment:
I'm late to the party, but it would be nice to see a "SVGs in IE don't have .children" (or alike) comment here. Otherwise a future developer may not know what this is for.